web/sites/default/config/views.settings.yml

```diff
@@ -9,7 +9,7 @@ ui:
     performance_statistics: false
     preview_information: true
     sql_query:
-      enabled: false
+      enabled: true
```

Comment:
I'm not sure we should be enabling sql_query all the way up to production, but we can pick this up in a future refactor.

Reply:
Sorry, this is a setting slipped in. I turned it on to see the query.

Reply:
I think I merged this before your additional commit. Let me see if I can still merge this change or if I need to have you submit another PR.

Reply:
Hmm. I'll need a separate PR for that last commit. My merge preceded it.